files/zh-cn/web/http/headers/expires/index.md

```diff
@@ -1,33 +1,51 @@
 ---
 title: Expires
 slug: Web/HTTP/Headers/Expires
+l10n:
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
 ---
 
-{{HTTPSidebar}}**`Expires`** 响应头包含日期/时间，即在此时候之后，响应过期。
-
-无效的日期，比如 0，代表着过去的日期，即该资源已经过期。
-
-如果在{{HTTPHeader("Cache-Control")}}响应头设置了 "max-age" 或者 "s-max-age" 指令，那么 `Expires` 头会被忽略。
-
-| Header type                                                               | {{Glossary("Response header")}} |
-| ------------------------------------------------------------------------- | ------------------------------- |
-| {{Glossary("Forbidden header name")}}                                     | no                              |
-| {{Glossary("Simple response header", "CORS-safelisted response-header")}} | yes                             |
+{{HTTPSidebar}}
+
+**`Expires`** 响应标头包含响应应被视为过期的日期/时间。
+
+无效的日期（比如 0）代表过去的日期，即该资源已经过期。
+
+> **备注：** 如果响应中有指令为 `max-age` 或 `s-maxage` 的 {{HTTPHeader("Cache-Control")}} 标头，则 `Expires` 标头会被忽略。
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th scope="row">标头类型</th>
+      <td>{{Glossary("Response header", "响应标头")}}</td>
+    </tr>
+    <tr>
+      <th scope="row">{{Glossary("Forbidden header name", "禁止修改的标头")}}</th>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th scope="row">
+        {{Glossary("CORS-safelisted response header", "列入 CORS 安全名单的响应标头")}}
```

Comment:
Please consider the translation of "CORS-safelisted response header":

- 列入 CORS 安全名单的响应标头
- CORS 安全的响应标头

Reply:
I think "列入 CORS 安全名单的响应标头" or "列入 CORS 安全列表的响应标头" is much more reliable. A header like "expires" was safe for CORS so they were listed in the safelist(whitelist) for some allowed strategy. The list is extendable, so I think it is better not to expand the interpretation so that the concept of the CORS-safelist will not be ignored. And out of accuracy consideration, a CORS-safe header may not listed in the CORS-safelist.